integration/features_test.go

```diff
@@ -445,8 +444,11 @@ func TestIntegration_WeiWatchers(t *testing.T) {
 
 func TestIntegration_MultiplierInt256(t *testing.T) {
 	app, cleanup := cltest.NewApplication()
+	eth := app.MockEthClient()
+	eth.Context("app.Start()", func(eth *cltest.EthMock) {
+	})
 	defer cleanup()
-	app.Start()
+	_ = app.Start()
```

Comment:
This was in response to a linter complaint about throwing away an `error`. Incidentally, for a number of other tests in this file, if you replace `app.Start()` with `assert.NoError(t, app.Start())`, you get failures due to unregistered mocks. I haven't fixed those up, since the mocks don't really seem relevant to the tests. But I think generally it's good to register that you're ignoring an error.